website/docs/getting-started/installation.mdx

```diff
@@ -20,42 +20,97 @@ import TOCInline from '@theme/TOCInline';
 <TOCInline toc={toc} />
 ```
 
-## Option A: Compiling Locally (Recommended)
+## Option A: Download Pre-Built Releases (Recommended)
 
-### Installing Dependencies
+### a) Pre-Built Binary
+
+1. Go to the [latest release on GitHub](https://github.com/moves-rwth/caesar/releases/latest) and download the release for your operating system.
+
+2. Extract the `caesar` binary from the downloaded archive.
+
+:::info Additional Steps
+
+<Tabs groupId="operating-systems">
+  <TabItem value="mac" label="macOS">
+
+  You may have to [release the binary from quarantine](https://support.apple.com/guide/mac-help/mh40616/mac) to allow execution:
+  ```bash
+  # Execute in the extracted folder
+  xattr -d com.apple.quarantine ./caesar
+  ```
+
+  </TabItem>
+  <TabItem value="debian" label="Debian/Linux">
+    No additional steps necessary.
+  </TabItem>
+  <TabItem value="win" label="Windows">
+    No additional steps necessary.
+  </TabItem>
+</Tabs>
+:::
+
+### b) Docker Image
+
+We also provide a Docker image.
+
+```bash
+docker pull ghcr.io/moves-rwth/caesar:latest
+```
+
+You can use this Docker image just like the `caesar` binary.
+To access files from your system you need to mount them as volumes.
+For example, if you want to verify a program in `./programs/example.heyvl`, the following command mounts the `./programs/` directory as `/root/caesar/programs/` inside the container and uses `caesar` to verify `example.heyvl`.
+
+```bash
+docker run -it -v $PWD/programs:/root/caesar/programs/ ghcr.io/moves-rwth/caesar:latest /root/caesar/programs/example.heyvl
+```
+
+## Option B: Compiling Locally
```

Comment:
Since this section is irrelevant to most people, could you add some vertical space before this heading? E.g. `<div style="margin-top: 2em" />`?

Reply:
I agree that there should be some space, but fail to see how this is related to this 'section being usually irrelevant'. I mean: it is a new section, you can clearly notice it by the bigger heading font. On my end it looks like there is not enough vspace due to the scrollbar above, but comparing with other pages of the documentation, the spacing is equivalent and I think a uniform design is good. Maybe one could/should handle this more generally in the site's CSS?

On a more general note: I find these straight-line (sub)subheadings without any indentation hard to navigate/group.